README.md

```diff
@@ -1,55 +1,118 @@
-git checkout <release tag>  // release tag that matches the branch id in the polkadot-deps
+# Switch into the Polkadot directory
+cd polkadot
+# Checkout the proper commit
+git checkout <release tag>  # release tag, just like in the example above: polkadot-0.9.18
 cargo build --release
+# If this fails due to  `feature `edition2021` is required` you have set an outdated rust default version. To fix this, you can simply run:
+cargo +nightly-2022-01-31 build --release # or any other up-to-date nightly version
 
 # Generate a raw chain spec
 ./target/release/polkadot build-spec --chain rococo-local --disable-default-bootnode --raw > rococo-local-cfde.json
+```
+Start the `Alice` validator:
+```bash
+./target/release/polkadot --chain rococo-local-cfde.json --alice --validator --tmp --port 30333 --ws-port 9944
+```
+When the node starts you will see several log messages. Take note of the node's Peer ID in the logs. We will need it when connecting other nodes to it. It will look something like this:
+```bash
+Local node identity is: 12D3KooWGjsmVmZCM1jPtVNp6hRbbkGBK3LADYNniJAKJ19NUYiq
+```
+Start the `Bob` validator (in a second terminal):
+```bash
+./target/release/polkadot --chain rococo-local-cfde.json --validator --bob --bootnodes /ip4/<Alice IP>/tcp/30333/p2p/<Alice Peer ID> --tmp --port 30334 --ws-port 9945
+```
+
+More information can be found in the Substrate tutorial [Start a Relay Chain](https://docs.substrate.io/tutorials/v3/cumulus/start-relay/)
+
+#### Reserve a para ID
+Go to [Polkadot Apps](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/parachains/parathreads). Register the parachain via
+Network > Parachains sub-page, click on Parathreads tab and use the + ParaId button. After registering, the collator should start producing blocks when the next era starts.
+![image](https://d33wubrfki0l68.cloudfront.net/ab3d311e37364a9706f2747b98b24fc259398152/2c4ba/static/4e9213b9ee2f65cc7fa9ccddd73679a3/c1b63/paraid-reserve.png)
 
-# Alice
-./target/release/polkadot --chain rococo-local-cfde.json --alice --tmp
 
-# Bob (In a separate terminal)
-./target/release/polkadot --chain rococo-local-cfde.json --bob --tmp --port 30334
-```
 
 #### Launch the Parachain
 
+Build the parachain:
 ```bash
 # Compile
-git clone https://github.com/scs/integritee-parachain.git
-git checkout master
+git clone https://github.com/integritee-network/parachain
+# Switch into the Integritee parachain directory & build it
+cd parachain
 cargo build --release
 
+```
+Generate custom parachain specification:
+```bash
+./target/release/integritee-collator build-spec --chain integritee-rococo-local-dev --disable-default-bootnode > integritee-rococo-local-dev.json
+```
+and update the `para_id` to the one you reserved on the relay-chain (default is `2000`):
+```json
+// --snip--
+  "para_id": 2000, // <--- your already registered ID
+  // --snip--
+      "parachainInfo": {
+        "parachainId": 2000 // <--- your already registered ID
+      },
+  // --snip--
```

Comment:
It is necessary to register, also the default. Yes it's 200 in the image, but I believe with the comment in the code it should be clear that you should take the one you have used in the reservation part. Do you agree?

Reply:
yes

Reply:
there is no choice here! it has to be the paraid which is hard-coded for that configuration (otherwise the parachain may produce blocks, but XCM will not work)

https://github.com/integritee-network/parachain/blob/616c31b2baa364c28daa43405342ea560a046cd2/polkadot-parachains/src/command.rs#L44-L48

Reply:
so 2000 it is!